### PR TITLE
Remove lists from Chits messages

### PR DESCRIPTION
**Pull request by StephenButtolph**
## Why this should be merged

We no longer send multiple chits over the wire. We also always send the last accepted block ID over the wire.

## How this works

Removes the repeated fields from the proto messages (this is backwards compatible)

## How this was tested

CI

### CODE REVIEW

**Review by gyuho — APPROVED** [review]
🎉

File: snow/engine/snowman/transitive.go [hunk 1]
```diff
@@ -295,8 +280,8 @@ func (t *Transitive) Chits(ctx context.Context, nodeID ids.NodeID, requestID uin
 }
 
 func (t *Transitive) QueryFailed(ctx context.Context, nodeID ids.NodeID, requestID uint32) error {
-	lastAccepted := t.acceptedFrontiers.AcceptedFrontier(nodeID)
-	if len(lastAccepted) == 1 {
+	lastAccepted, ok := t.acceptedFrontiers.LastAccepted(nodeID)
+	if ok {
 		// Chits calls QueryFailed if [votes] doesn't have length 1, so this
```
Comment (darioush) [inline snow/engine/snowman/transitive.go:285]: probably should update this comment

File: snow/engine/snowman/transitive_test.go [hunk 2]
```diff
@@ -343,7 +337,7 @@ func TestEngineQuery(t *testing.T) {
 			t.Fatalf("Asking for wrong block")
 		}
 	}
-	if err := te.Chits(context.Background(), vdr, *queryRequestID, []ids.ID{blk1.ID()}, nil); err != nil {
+	if err := te.Chits(context.Background(), vdr, *queryRequestID, blk1.ID(), blk1.ID()); err != nil {
```
Comment (darioush) [inline snow/engine/snowman/transitive_test.go:340]: why did we go from passing `nil` to accepted to passing the block ID?

**Review by darioush — APPROVED** [review]

File: snow/engine/snowman/transitive_test.go (hunk 2 above)
Comment (StephenButtolph) [reply snow/engine/snowman/transitive_test.go:340]: This change requires the second field to be populated

File: snow/engine/snowman/transitive.go (hunk 1 above)
Comment (StephenButtolph) [reply snow/engine/snowman/transitive.go]: removed the comment

**Pull request merged**